packages/hawtio/src/plugins/runtime/runtime-service.ts

```diff
@@ -0,0 +1,134 @@
+import { jolokiaService } from '@hawtiosrc/plugins/shared'
+import { Metric, SystemProperty, Thread } from './types'
+
+export const REFRESH_INTERVAL = 5000
+function convertMsToDaysHours(ms: number): string {
+  const seconds = Math.floor(ms / 1000)
+  const minutes = Math.floor(seconds / 60)
+  const hours = Math.floor(minutes / 60)
+  const days = Math.floor(hours / 24)
+
+  return `${days} days, ${hours % 24} hours`
+}
+
+export function getSystemProperties(): Promise<SystemProperty[]> {
```

Comment:
I'd use `load` instead of `get` when a function reads data from an external service like jolokia.